db/log_reader.h

```diff
@@ -170,6 +185,10 @@ class Reader {
   // is only for WAL logs.
   UnorderedMap<uint32_t, size_t> recorded_cf_to_ts_sz_;
 
+  // if log reader is skipped, may need to drop bytes
+  // until seek to the first of a record
+  bool skipped_;
```

Comment:
Consider changing the name to something like `skip_first_fragmented_record_` since this field will be set to false after skipping a record.

Reply:
👌